homeassistant/components/lovelace/cast.py

```diff
@@ -112,7 +116,7 @@ async def async_browse_media(
 
 async def async_play_media(
     hass: HomeAssistant,
-    cast_entity_id: str,
+    cast_entity_id: MediaType | str,
     chromecast: Chromecast,
     media_type: str,
```

Comment:
This does look wrong. Are you sure it's correct?

Reply:
Doh! fixed.